src/pkg/backup/details/details.go

```diff
@@ -388,6 +388,7 @@ func (d *Details) add(
 		ShortRef:    repoRef.ShortRef(),
 		ParentRef:   repoRef.ToBuilder().Dir().ShortRef(),
 		LocationRef: locationRef.String(),
+		ItemRef:     repoRef.Item(),
```

Comment:
I think this is still going to generate ItemRefs with suffixes for OD/SP (e.x. `<item ID>.data`). I thought we were aiming for just the item ID without any suffixes?